.github/workflows/pull_request.yml

```
@@ -57,16 +58,13 @@ jobs:
     needs: yaml-check
     if: ${{needs.yaml-check.outputs.toggle_spell_check == 'yes'}}
     uses: jhudsl/ottr-reports/.github/workflows/main.yml@acoffman/artifacts
```

```suggestion
    uses: jhudsl/ottr-reports@acoffman/artifacts
```

Should it just be calling the action and not the main.yml ?

---

@cansavvy Yep! I believe this is actually the root of the problem and the indirection with the main.yml is what's causing the double checkout issue. If I called the action directly, I don't think we'd see any of this. 

I could totally be missing something, but at least when I was trying to do it that way, it didn't seem to want to let me use it as an action directly unless it was "published" checking it out and using it as a local action was the only way I could figure out how to use the unpublished branch version. So, that's why I was doing it this strange way for testing. If you know how to call it the "right" way as an action, but point to the branch instead, that would be awesome!

---

I got this part fixed! But now have a new token-related problem lol https://github.com/ottrproject/ottr-reports/pull/3#issuecomment-2776215736 Any ideas? @acoffman